package.json

```diff
@@ -34,20 +34,25 @@
     "test:watch": "npm run test -- --watch",
     "generate-component-install": "cd component-generator && npm install",
     "generate-component": "cd component-generator && npm start",
-    "generate-component-lint": "cd component-generator && npm run lint"
+    "generate-component-lint": "cd component-generator && npm run lint",
+    "type-check": "tsc --noEmit",
+    "type-check:watch": "npm run type-check -- --watch",
+    "build-types": "tsc --emitDeclarationOnly"
   },
   "dependencies": {
     "@fortawesome/fontawesome-svg-core": "^1.2.36",
     "@fortawesome/react-fontawesome": "^0.1.18",
     "@popperjs/core": "^2.11.4",
+    "@types/enzyme": "^3.10.12",
+    "@types/react-test-renderer": "^18.0.0",
     "airbnb-prop-types": "^2.12.0",
     "bootstrap": "^4.6.1",
     "classnames": "^2.3.1",
     "email-prop-type": "^3.0.0",
     "font-awesome": "^4.7.0",
     "lodash.uniqby": "^4.7.0",
     "mailto-link": "^1.0.0",
-    "prop-types": "^15.8.1",
+    "prop-types": "^15.7.2",
```

Comment:
[curious] Why is this downgraded?

Reply:
Perhaps this is due to the Node js version